tests/test.js

```diff
@@ -3,28 +3,41 @@ import { readdirSync, readFileSync } from "node:fs";
 import path from "node:path";
 import { JSDOM } from "jsdom";
 import { fileURLToPath } from "url";
+import { execa } from "execa";
 
 const __dirname = fileURLToPath(new URL(".", import.meta.url));
 
 describe("Embroider build", function () {
   let indexHtml;
 
-  const scenarios = readdirSync(path.join(__dirname, "..", "test-apps"));
+  const scenarios = readdirSync(path.join(__dirname, "..", "test-apps")).filter(
+    (file) =>
+      [
+        // skip these test apps
+        "external-resource-missing-integrity-hash",
+        "external-resource-with-integrity-hash",
```

Comment:
Assuming you mean the `external-resource-with-integrity-hash` app?

I didn't feel it offered value to run the normal tests against these apps as they only differ from default in that they include an external resource. Everything else is exactly the same. But I can make the change if you think it's safer to still test them.

> Maybe we could prevent building the app twice by building all apps before running any test? Or we could keep a list, which app has been build already and skip if it exists in that list.

This is a good idea.

Reply:
> I didn't feel it offered value to run the normal tests against these apps as they only differ from default in that they include an external resource. Everything else is exactly the same. But I can make the change if you think it's safer to still test them.

Not sure how likely it is that we introduce a bug which would be caught by this. But one theoretical bug would be looping over resources and breaking entire loop (`break` instead of `continue`) when an external resource is detected. In that case also other resources defined later would not be processed correctly.